Dockerfile

```diff
@@ -11,61 +11,58 @@ LABEL \
     description="Mobile Security Framework is an intelligent, all-in-one open source mobile application (Android/iOS/Windows) automated pen-testing framework capable of performing static, dynamic analysis and web API testing"
 
 #Environment vars
-ENV DEBIAN_FRONTEND="noninteractive"
-ENV PDFGEN_PKGFILE="wkhtmltox_0.12.5-1.bionic_amd64.deb" 
-ENV PDFGEN_URL="https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/${PDFGEN_PKGFILE}"
-ENV JDK_FILE="openjdk-12_linux-x64_bin.tar.gz"
-ENV JDK_URL="https://download.java.net/java/GA/jdk12/GPL/${JDK_FILE}"
+ENV DEBIAN_FRONTEND="noninteractive" \
+    JDK_FILE="openjdk-12_linux-x64_bin.tar.gz" \
+    WKH_FILE="wkhtmltox-0.12.5-dev-163e124_linux-generic-amd64.tar.xz"
+
+ENV JDK_URL="https://download.java.net/java/GA/jdk12/GPL/${JDK_FILE}" \
+    WKH_URL="http://www.ajvg.com/downloads/${WKH_FILE}"
```

Comment:
Which is this source?
Is it official?

Reply:
that's a patched version made to reduce wkhtml problem , it will be the same if we switch to  Alpine we will need to use a patched version .
We can stay on official version, on this case I will back port Dockerfile optimisation to the version with official wkhtmltopdf  but we won't see big improvement in size for the image

Reply:
not an official but static compiled and patched regarding QT5

Reply:
That's okey, but we stick to official builds as much as possible.

Reply:
will see if I can symlink instead of making copy